php/libraries/LorisForm.class.inc

```diff
@@ -373,7 +372,7 @@ class LorisForm
                 $optionKey = (string) $optionKey;
             }
 
-            if ($optionKey === $this->getValue($el['name'])) {
+            if ($optionKey === $el['selected']) {
```

Comment:
I think there's a different pull request for this change, so it shouldn't be part of this one (unless it depends on the change, in which case we should wait until that one finishes getting reviewed/merged..)